src/block/blog-posts/taxonomy-control.js

```diff
@@ -145,12 +147,30 @@ class TaxonomyControl extends Component {
 					/>
 				}
 				{ taxonomyTypeOptions.length > 0 &&
+				<Fragment>
 					<AdvancedSelectControl
+						label={ __( 'Taxonomy Filter Type', i18n ) }
+						options={ [
+							{ label: 'Included In', value: '__in' },
```

Comment:
Missing translation function. Please also fix the next line

```suggestion
							{ label: __( 'Included In', i18n ), value: '__in' },
```